01_fetch/src/fetch_noaa_nerrs.py

```diff
@@ -0,0 +1,51 @@
+import pandas as pd
+import os 
+import numpy as np
+import urllib
+import requests, json
+
+def fetch_metadata(station_id, metadata_outfile):
+    '''fetch metadata for NOAA NERRS station'''
```

Comment:
Is this NERRS data? I was thinking this product was call Currents or something because I had been looking at NERRS

Reply:
Yes your right! It is definitely not NERRS data. So, it is a little confusing; NOS and current data seem a little ambiguous. I think the current data comes from the National Ocean Service (NOS). I will confirm this though.